packages/terra-time-input/src/TimeInput.module.scss

```diff
@@ -43,11 +47,7 @@
       .time-input-hour,
       .time-input-minute,
       .time-input-second {
-        color: var(--terra-time-input-hover-color, #1c1f21);
-      }
-
-      .meridiem-display {
-        color: var(--terra-time-input-hover-color, #1c1f21);
+        color: var(--terra-time-input-time-component-hover-color, #1c1f21);
```

Comment:
Why the addition of the word `component` in the variable name?

Reply:
At present, same variables are used for the sub-components (hour, minute, second) as well as their wrapper while they are technically different entities; although the variables are the same, the difference in how these entities are presented is currently brought out by the [explicit styles laid out under `.desktop` selector](https://github.com/cerner/terra-framework/blob/a9e915c6dee0e4cecf36eac1882e29282f13a342/packages/terra-time-input/src/TimeInput.module.scss#L161-L193). 

This gets confusing when there is no concept of mobile v/s desktop anymore (both hit the same code with this revamp), so I introduced the word 'time-component' into the variables that are meant to style the sub-components (hour, minute, second) _only_. This, imo, made it more readable and made it a lot easier to differentiate these components in the theming repositories downstream as well.

Reply:
Sounds good, thanks for the clarification @nramamurth